### PR TITLE
replace LLL in transacton data

### PR DESCRIPTION
**Pull request by winsvega**
replace LLL code in transaction data field

### CODE REVIEW

File: test/tools/libtesteth/ImportTest.cpp [hunk 1]
```diff
@@ -265,12 +265,16 @@ json_spirit::mObject ImportTest::makeAllFieldsHex(json_spirit::mObject const& _i
...
+        bool isData = false;
+        std::string key = i.first;
 
-		if (key == "data" || key == "extraData")
-			continue;
+        if (key == "extraData")
+            continue;
+
+        if (key == "data")
+            isData = true;
```
Comment (pirapira) [inline test/tools/libtesteth/ImportTest.cpp]:
```
bool isData = (key == "data");
```
and you can remove two other lines.

File: test/tools/libtesteth/TestHelper.cpp [hunk 2]
```diff
@@ -235,29 +235,33 @@ bytes importData(json_spirit::mObject const& _o)
 {
     bytes data;
     if (_o.at("data").type() == json_spirit::str_type)
-        data = importByteArray(_o.at("data").get_str());
+        data = importByteArray(replaceLLL(_o.at("data").get_str()));
     else
         for (auto const& j : _o.at("data").get_array())
-            data.push_back(toByte(j));
+            data.push_back(toByte(replaceLLL(j.get_str())));
```
Comment (pirapira) [inline test/tools/libtesteth/TestHelper.cpp]: What is happening here?  `toByte(...)` is just one byte, no?

File: test/tools/libtesteth/TestHelper.cpp [hunk 3]
```diff
@@ -235,29 +235,33 @@ bytes importData(json_spirit::mObject const& _o)
...
     return data;
 }
 
+string replaceLLL(string const& _code)
+{
+    string compiledCode;
+    compiledCode = compileLLL(_code);
```
Comment (pirapira) [inline test/tools/libtesteth/TestHelper.cpp]: `string const compiledCode = compileLLL(_code);`
should do it.

**Comment by pirapira** [comment]
@chfast can you guess what's happening here?  In CircleCI Mac build:

```
==> Uploading
    .url https://codecov.io
    .query service=circleci&package=py2.0.11&yaml=codecov.yml&job=2049.0&build=2049.0&branch=lll&commit=22110560b5107497a72d0888b70fa7b8d8f128e8&slug=ethereum%2Fcpp-ethereum
    Pinging Codecov...
    Uploading to S3...
Error: 'ascii' codec can't decode byte 0xe2 in position 12947300: ordinal not in range(128)
```

**Comment by chfast** [comment]
I'm inspecting the problem, see https://github.com/codecov/codecov-python/pull/121 and https://github.com/codecov/codecov-python/issues/130.

You can merge with this error unless you need a code coverage report.

**Pull request merged**